packages/aws-cdk/lib/init-templates/app/csharp/dotnet.template.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
+
+rem   lets dotnet operate on the Visual Studio project from the CDK app's main directory
+rem   without explicitly having to specify the path to the .csproj file
+
+cd "src\%name.PascalCased%"     &   rem change to the directory containing .csproj
+dotnet.exe %*                   &   rem execute dotnet command from there
```

Comment:
how will this file be used on non-windows operating systems? or does it only apply to windows?